lib/openid_connect.ex

```diff
@@ -264,11 +264,13 @@ defmodule OpenIDConnect do
   end
 
   defp discovery_document(provider, name) do
-    GenServer.call(name, {:discovery_document, provider})
+    {:ok, doc} = GenServer.call(name, {:discovery_document, provider})
+    doc
   end
 
   defp jwk(provider, name) do
-    GenServer.call(name, {:jwk, provider})
+    {:ok, jwk} = GenServer.call(name, {:jwk, provider})
+    jwk
```

Comment:
Doesn't this make this code *more* brittle? If the `GenServer.call` call returns an error, this code will now raise an error instead of return the `{:error, reason}` tuple as it did before.

Don't the *callers* of these functions need to be updated to handle possible errors instead?

Reply:
@kenny-evitt, you're right, at least partially. This particular part of the change can be done better. I will push an additional commit to make some improvements.

Reply:
@kenny-evitt perhaps I should have been clear that I had addressed this comment on my branch.

This is now 3 years old and I'd like to get it merged. It'll need a rebase. My apologies for letting it sit so long, but I think this is important to merge as this library is basically unusable in our environment without this error-handling fix.

Reply:
@nbryant42 Apparently [this PR](https://github.com/DockYard/openid_connect/pull/50) fixes/'closes' the issue. That PR was just merged two weeks ago.